packages/typedoc-plugin-appium/resources/templates/commands.hbs

```diff
@@ -0,0 +1,39 @@
+{{> header}}
+
+{{#ifShowPageTitle}}
+
+# {{{reflectionTitle true}}}
+
+{{/ifShowPageTitle}}
+
+{{#with model}}
+
+{{#if hasComment}}
+
+{{> comment}}
+
+{{/if}}
+
+{{/with}}
+
+{{#with model}}
+
+{{#if hasRoute}}
+## {{pluralize 'Route' routeCount}}
```

Comment:
maybe `Command` or `Driver Command` or `API Command` or something? dunno if `Routes` is going to do it for most users' understanding of what these things are.

Reply:
right.  route is technical jargon.